src/containers/Playlist/PlaylistContainer.tsx

```diff
@@ -38,10 +40,10 @@ const PlaylistContainer = ({ playlistId, onPlaylistUpdate, style, children, show
     if (playlist && onPlaylistUpdate) onPlaylistUpdate(playlist);
   }, [playlist, onPlaylistUpdate]);
 
-  if (playlistId === PersonalShelf.Favorites) playlist = favoritesPlaylist;
-  if (playlistId === PersonalShelf.ContinueWatching) playlist = watchHistoryPlaylist;
+  if (type === PersonalShelf.Favorites) playlist = favoritesPlaylist;
+  if (type === PersonalShelf.ContinueWatching) playlist = watchHistoryPlaylist;
 
-  if (!playlistId) return <p>No playlist id</p>;
+  if (!playlistId && !type) return <p>No playlist id and type</p>;
```

Comment:
It's probably better to throw an error here, so that a company knows when the config is messed up (a warning message in the html might get overlooked)